internal/network/manifests_generator_test.go

```diff
@@ -305,3 +305,52 @@ var _ = Describe("dnsmasq manifest", func() {
 	})
 
 })
+
+var _ = Describe("telemeter manifest", func() {
+
+	var (
+		ctx                   = context.Background()
+		log                   *logrus.Logger
+		ctrl                  *gomock.Controller
+		mockManifestsApi      *mocks.MockManifestsAPI
+		manifestsGeneratorApi ManifestsGeneratorAPI
+		db                    *gorm.DB
+		dbName                string
+		clusterId             strfmt.UUID
+		cluster               common.Cluster
+	)
+
+	BeforeEach(func() {
+
+		log = logrus.New()
+		ctrl = gomock.NewController(GinkgoT())
+		mockManifestsApi = mocks.NewMockManifestsAPI(ctrl)
+		manifestsGeneratorApi = NewManifestsGenerator(mockManifestsApi, Config{ServiceBaseURL: "https://api.stage.openshift.com"})
+		db, dbName = common.PrepareTestDB()
+		clusterId = strfmt.UUID(uuid.New().String())
+
+		cluster = common.Cluster{
+			Cluster: models.Cluster{
+				ID: &clusterId,
+			},
+		}
+		Expect(db.Create(&cluster).Error).NotTo(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+		common.DeleteTestDB(db, dbName)
+	})
+
+	It("happy flow", func() {
```

Comment:
Please add a test case with  ServiceBaseURL that doesn't have the word `stage`

Reply:
Done.

Reply:
Where is the test?